sources/views/data.js

```diff
@@ -0,0 +1,46 @@
+import { JetView } from "webix-jet";
+import { countries } from "../models/countries.js";
+import { statuses } from "../models/statuses.js";
+
+import DataTable from "./dataTable.js";
+
+export default class Data extends JetView {
+	config() {
+		const dataMenu = {
+			view: "segmented",
+			localId: "dataMenu",
+			value: "countries",
+			options: [
+				{
+					value: "Countries",
+					id: "countries"
+				},
+				{
+					value: "Statuses",
+					id: "statuses"
+				}
+			]
+		};
+		const multiView = {
+			cells: [
+				{
+					id: "countries",
+					rows: [new DataTable(this.app, "", countries)]
+				},
+				{
+					id: "statuses",
+					rows: [new DataTable(this.app, "", statuses)]

```

Comment:
Use localId insted of regular id

Reply:
+ 1

provide localIds for Webix views and normal ids for segmented options

Reply:
Ready to review.